src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -22,12 +22,32 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// </summary>
     internal sealed partial class NullableWalker : DataFlowPassBase<NullableWalker.LocalState>
     {
+        // PROTOTYPE(NullableReferenceTypes): Reference the collections directly from the NullableWalker
+        // for the containing method rather than copying the collections. (Items are added to the collections
```

Comment:
>for the containing method [](start = 11, length = 25)

There are a lot of hidden assumptions in this comment. e.g. what "containing method"? I suspect you could say somthing like this data is used to communicate the state during analysis of a method to its nested methods (lambdas and local functions) when analyzing them. But I'm just guessing.